fieldservice_account/models/fsm_order.py

```diff
@@ -32,31 +33,45 @@ class FSMOrder(models.Model):
                                      default='draft')
     bill_to = fields.Selection([('location', 'Bill Location'),
                                 ('contact', 'Bill Contact')],
-                               string="Bill to",
+                               string='Bill to',
                                required=True,
-                               default="location")
+                               default='location')
     customer_id = fields.Many2one('res.partner', string='Contact',
                                   domain=[('customer', '=', True)],
                                   change_default=True,
                                   index=True,
                                   track_visibility='always')
+    cost_method = fields.Selection([('timesheet', 'Timesheets'),
+                                    ('fixed', 'Fixed Rate')],
+                                   string='Cost Method',
+                                   required=True,
+                                   default='timesheet')
+    fixed_cost = fields.Float(string='Fixed Cost')
```

Comment:
@brian10048 Why not use the product below to get the costing method from the product category and the standard cost from the product?

Reply:
... and apply Price Lists to get the amount to bill.

Reply:
@max3903 @dreispt - That makes more sense

Your suggestions have been made.